plugins/oauth/components/AuthenticatedUserAccountDropdown.js

```diff
@@ -23,6 +41,11 @@ export default class AuthenticatedUserAccountDropdown extends UserAccountDropdow
         id: "header-user-label",
         selectable: false
       },
+      {
+        html: `${Languages[UserLanguageStore.get()]} (${i18nMark("Change")})`,
```

Comment:
"Change" is marked but isn't translated. I think this needs to be translated via macro:

```suggestion
        html: <Trans render="span">{Languages[UserLanguageStore.get()]} (Change)</Trans>,
```
also add the import

`import { Trans } from "@lingui/macro"`